TwilioVerifySDKTests/Plans/IntegrationSuite.xctestplan

```diff
@@ -39,8 +39,6 @@
         "PushChallengeProcessorTests",
         "PushFactoryTests",
         "RequestHelperTests",
-        "SecureStorageTests",
```

Comment:
Why was this removed?

Reply:
That test is part of the ignore list for the Integration Pipeline. It was failing because the secure storage tests didn’t cover all the relevant code, resulting in lower code coverage.